packages/sri-lanka/README.md

```diff
@@ -0,0 +1,52 @@
+# 🇱🇰 Sri Lanka
```

Comment:
Could you also create a `package.json` file like the other maps please? I forgot to write it in the [contribution guide](https://github.com/VictorCazanave/svg-maps/blob/master/CONTRIBUTING.md) 🙇

Reply:
I will do it 👍

Reply:
@VictorCazanave i added the package.json file ;-).